### PR TITLE
Release 0.9.12

### PR DESCRIPTION
**Pull request by mpranj**
## Basics

- [ ] Short descriptions of your changes are in the release notes
      (added as entry in `doc/news/_preparation_next_release.md` which
      contains `_(my name)_`)
      **Please always add something to the release notes.**
- [ ] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildservers are happy. If not, fix **in this order**:
  - [ ] add a line in `doc/news/_preparation_next_release.md`
  - [ ] reformat the code with `scripts/dev/reformat-all`
  - [ ] make all unit tests pass
  - [ ] fix all memleaks
- [ ] The PR is rebased with current master.

## Checklist

- [ ] I added unit tests for my code
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation (see [Documentation Guidelines](https://www.libelektra.org/devgettingstarted/documentation))
- [ ] I fixed all affected decisions (see [Decision Process](https://www.libelektra.org/decisions/decision-process))
- [ ] I added code comments, logging, and assertions as appropriate (see [Coding Guidelines](https://www.libelektra.org/devgettingstarted/coding))
- [ ] I updated all meta data (e.g. README.md of plugins and [METADATA.ini](https://master.libelektra.org/doc/METADATA.ini))
- [ ] I mentioned [every code](/.reuse/dep5) not directly written by me in [reuse syntax](https://reuse.software/)

## Review

- [ ] Documentation is introductory, concise, good to read and describes everything what the PR does
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to [our Coding Guidelines](https://master.libelektra.org/doc/CODING.md)
- [ ] APIs are conforming to [our Design Guidelines](https://master.libelektra.org/doc/DESIGN.md)
- [ ] Code is consistent to [our Design Decisions](https://master.libelektra.org/doc/decisions)

## Labels

- [ ] Add the "work in progress" label if you do not want the PR to be reviewed yet.
- [ ] Add the "ready to merge" label **if the basics are fulfilled** and no further pushes are planned by you.

### CODE REVIEW

**Review by markus2330 — APPROVED** [review]
Great job, looking forward to the release!

**Comment by mpranj** [comment]
Need to sort out some details still 😁

**Comment by mpranj** [comment]
https://github.com/ElektraInitiative/libelektra/blob/df855e37a52e7a371234a42328f5914406883b70/doc/news/_preparation_next_release.md?plain=1#L119

@kodebach do you want to add some more notes regarding the new backend for this release or should we remove/postpone this note?

**Comment by kodebach** [comment]
The new `kdb mount` isn't done yet, @flo91 is still working on it, so just remove the `TODO` for now`. The rest of the `new-backend` details should be fine as-is.

**Comment by mpranj** [comment]
@kodebach @atmaxinger @markus2330 If you have time, take a look at the release notes (9e886a06d8c077646ad8e746cf11eba08d875472 or the complete file).

If there are no comments, I'll try release tomorrow (maybe with a test-run today).

File: doc/news/2021-06-07_0.9.6.md [hunk 1]
```diff
@@ -223,7 +223,7 @@ _(Michael Tucek)_
 - We added release images that come with pre-installed dependencies and sudo permissions for each distribution we build packages for. _(Robert Sowula)_
 - Use Clang 12 and Gradle 7.0 in Debian Sid image. _(Mihael Pranjić)_
 - Remove Boost and some unused dependencies from all Docker images. _(Mihael Pranjić)_
-- Use Gradle 7.0 and Ronn-NG `0.10.1.pre1` in Docker images. _(Mihael Pranjić)_
+- Use Gradle 7.0 and Ronn-NG `0.10.1.pre3` in Docker images. _(Mihael Pranjić)_
```
Comment (kodebach) [inline doc/news/2021-06-07_0.9.6.md:226]: Was this change intentional?

**Review by kodebach — APPROVED** [review]
LGTM

File: doc/news/2021-06-07_0.9.6.md (hunk 1 above)
Comment (mpranj) [reply doc/news/2021-06-07_0.9.6.md:226]: No, good catch! ❤️

File: doc/news/_preparation_next_release.md [hunk 2]
```diff
@@ -331,384 +249,181 @@ This section keeps you up-to-date with the multi-language support provided by El
...
-- <<TODO>>
-- <<TODO>>
-- Restructured decisions directories based on new agreed-upon [steps](../decisions/STEPS.md) _(Maximilian Irlinger @atmaxinger)_
-- <<TODO>>
-- <<TODO>>
-- Decision for [types of `KeySet`s](../decisions/2_solutions_clear/keyset_types.md) _(@kodebach)_
-- <<TODO>>
-- Added [Documentation Guidelines](https://master.libelektra.org/doc/contrib/documentation.md) _(Markus Raab)_
+- Update [internal cache](../decisions/4_decided/internal_cache.md). _(Markus Raab)_
+- Move [internal cache](../decisions/4_decided/internal_cache.md) back to draft. _(@kodebach)_
```
Comment (atmaxinger) [inline doc/news/_preparation_next_release.md:365]: Are those two not redundant? It's now in `decided`.

File: doc/news/_preparation_next_release.md [hunk 3]
```diff
@@ -193,75 +170,54 @@ The following text lists news about the [plugins](https://www.libelektra.org/plu
...
 
-- Made logchange a notification-send hook plugin _(Maximilian Irlinger @atmaxinger)_
-
-### website
-
-- Fix broken `/pythongen` link on homepage _(@stefnotch)_
-- Fix redirect logic to not cause loops _(@stefnotch)_
-- Remove duplicated link to `TESTING.md` file _(@stefnotch)_, _(@janldeboer)_
-- Restructure parts of the links on the website _(@stefnotch)_, _(@janldeboer)_
-- Removed broken links to packages for Linux distributions. _(@Dynamichost96)_
```
Comment (atmaxinger) [inline doc/news/_preparation_next_release.md:204]: Why are all the lines above removed? Are they leftovers from a previous version?

Comment (mpranj) [reply doc/news/_preparation_next_release.md:204]: They were moved from the plugin section to the website section below ...

File: doc/news/_preparation_next_release.md (hunk 2 above)
Comment (mpranj) [reply doc/news/_preparation_next_release.md]: You're right, looks weird for the final release notes.

Both entries were probably true at some point in time.

**Comment by mpranj** [comment]
Thank you all for the reviews ❤️
